draft-ietf-quic-recovery.md

```diff
@@ -850,30 +850,26 @@ the reduction to once per round trip.
 
 ## Tail Loss Probe
 
-If recovery sends a tail loss probe, no change is made to the congestion
-window or pacing rate.  Acknowledgement or loss of tail loss probes are
-treated like any other packet.
+If recovery sends a tail loss probe, no change is made to the congestion window.
+Acknowledgement or loss of tail loss probes are treated like any other packet.
 
 ## Retransmission Timeout
 
-When retransmissions are sent due to a retransmission timeout alarm, no
-change is made to the congestion window or pacing rate until the next
-acknowledgement arrives.  When an ack arrives, if packets prior to the first
-retransmission timeout are acknowledged, then the congestion window
-remains the same.  If no packets prior to the first retransmission timeout
-are acknowledged, the retransmission timeout has been validated and the
-congestion window must be reduced to the minimum congestion window and
-slow start is begun.
-
-## Pacing Rate
-
-The pacing rate is a function of the mode, the congestion window, and
-the smoothed rtt.  Specifically, the pacing rate is 2 times the
-congestion window divided by the smoothed RTT during slow start
-and 1.25 times the congestion window divided by the smoothed RTT during
-congestion avoidance.  In order to fairly compete with flows that are not
-pacing, it is recommended to not pace the first 10 sent packets when
-exiting quiescence.
+When retransmissions are sent due to a retransmission timeout alarm, no change
+is made to the congestion window until the next acknowledgement arrives.  When
+an ack arrives, if packets prior to the first retransmission timeout are
+acknowledged, then the congestion window remains the same.  If no packets prior
+to the first retransmission timeout are acknowledged, the retransmission timeout
+has been validated and the congestion window must be reduced to the minimum
+congestion window and slow start is begun.
```

Comment:
nit: begun is weird in retrospect.  Maybe re-entered?

Reply:
reworded paragraph